packages/nimble-components/src/table-column/icon/tests/table-column-icon.stories.ts

```diff
@@ -40,6 +40,12 @@ const simpleData = [
         lastName: 'Simpson',
         status: 'success',
         isChild: true
+    },
+    {
+        firstName: 'John',
+        lastName: 'Doe',
```

Comment:
```suggestion
        firstName: 'Abbey',
        lastName: '?',
```

https://simpsonswiki.com/wiki/Abbey